backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -61,6 +66,18 @@ private async void OnMissionUpdate(object? sender, MqttReceivedArgs mqttArgs)
                     mission.Status,
                     mission.RobotId
                 );
+            if (mission.Status == "completed")
```

Comment:
Use proper string comparison pattern "string.Equals(otherString, StringComparison.OrdinalIgnoreCase)" (This should probably be case insensitive)